src/sidebar/services/test/root-thread-test.js

```diff
@@ -165,9 +162,7 @@ describe('rootThread', function() {
     });
 
     it('passes the highlighted set to buildThread()', function() {
-      fakeStore.state = Object.assign({}, fakeStore.state, {
-        highlighted: ['id1', 'id2'],
-      });
+      fakeStore.state.selection.highlighted = ['id1', 'id2'];
```

Comment:
I did this in a number of places. In the same file I see that we set the store directly and we also set it as a copy which was not consistent. Since there is no possible way to mutate the store across test cases Its simpler to just set the values directly rather than copy to new objects .